src/index.css

```diff
@@ -0,0 +1,164 @@
+* {
```

Comment:
Please be sure that your app works for mobile sizes 🤓 As the requirements mentioned "mobile web application", having the design work for mobile devices is key 📝

Your design on mobile 
![image](https://user-images.githubusercontent.com/15807343/189495298-9f0bcc02-6ed6-4c8b-adc5-ff58ca4dabeb.png)